torch/autograd/variable.py

```diff
@@ -332,6 +332,11 @@ def multinomial(self, num_samples=1, replacement=False):
     def bernoulli(self):
         return Variable(torch.bernoulli(self.data))
 
+    def standard_gamma(self, grad=None):
+        if grad is None:
+            return Variable(torch.standard_gamma(self.data))
+        return Variable(torch.standard_gamma(self.data, grad), requires_grad=self.requires_grad)
```

Comment:
You should just use the Function subclass you implemented here.

Reply:
Thanks for the help! Should I embed that subclass in `torch.autograd.variable`, or should `torch.distributions` be the canonical interface for `standard_gamma()` for `Variable`s?